samples/csharp/MicrosoftEduGraphSamples/Utilities/GlobalMethods.cs

```diff
@@ -44,9 +44,13 @@ public static void ValidateConfiguration(IConfiguration configuration)
             {
                 throw new Exception("Missing studentAccount please check appconfig.json file.");
             }
-            else if (string.IsNullOrEmpty(_config["password"]))
+            else if (string.IsNullOrEmpty(_config["teacherpassword"]))
```

Comment:
check all samples where you are using `teacherpassword`, and use "P" -> `teacherPassword`

Reply:
updated teacherpassword to teacherPassword for all instances and checked all methods of all files.